sentry_streams/sentry_streams/examples/transformer.py

```diff
@@ -58,12 +59,12 @@ def merge(self, other: Self) -> Self:
         name="myinput",
         stream_name="events",
     )
-    .apply("mymap", Map(function=parse))
+    .apply("parser", Parser(deserializer=json_parser))
```

Comment:
This gives the user flexibility to defer deserializng the message, but I'd like to maybe find a way to do it where the platform figures out where to do the parsing based on some info the user provides.